src/Components/Beers.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+import Nav from './Nav';
+
+class Beers extends Component {
+  state = {
+    beerList: [],
+  };
+
+  componentDidMount = async () => {
+    const ApiBeers = await axios.get('https://api.punkapi.com/v2/beers');
+    this.setState({ beerList: ApiBeers.data });
+    console.log(ApiBeers.data)
```

Comment:
Remember to remove unnecessary `console.logs` after you're done testing that everything works.